libs/core/src/stories/Welcome.stories.mdx

```diff
@@ -3,14 +3,21 @@ import { Meta, Story } from '@storybook/addon-docs';
 
 ## Foundations
-Our system starts with some core visual elements: Type, Color, and Icons. We also prescribe common practices for aspects of the system.
+Our design system is built upon a foundation of three core visual elements: Typography, Color, and Icons. These fundamental components work harmoniously to create a consistent and cohesive visual language throughout our designs.
 
 ## Components
-Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.
-Code examples, property options, and best practices can all be discovered here. Links to Figma design documentation can also be found in this section.
+Components are the backbone of our design system, acting as the building blocks for crafting seamless and cohesive user interfaces. Our components are available in TypeScript and React, which maximizes usability and ensure compatibility with different development environments.
+
+Our extensive documentation provides code examples, property options, and best practices for each component, making it easy for designers and developers to understand and implement them effectively. This thorough resource empowers teams to create consistent, high-quality experiences while reducing the learning curve and boosting productivity.
+
+In addition to the technical documentation, we also provide links to Figma design files for each component. These design assets visually represent the components and their various states, allowing designers to incorporate them into their mockups and prototypes easily. By integrating code and design resources, our design system is a comprehensive, one-stop solution for creating outstanding experiences for all Kajabi entrepreneurs.
 
 ## Contributing
-We encourage everyone to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
+We firmly believe in fostering an inclusive and collaborative environment where everyone feels empowered to participate in the evolution of our design system. The success and longevity of the system depend on the collective efforts and insights of our diverse team, as well as the users who interact with it daily.
+
+We can continuously refine and expand our design system to meet evolving needs and expectations by encouraging open communication, feedback, and contributions from all stakeholders. This collaborative approach helps identify potential improvements and new ideas and promotes a shared sense of ownership and investment in the system's success.
+
+To facilitate this process, we provide [resources and guidelines](/docs/resources-contributing-to-sage--page) on understanding, using, and contributing to the design system. By offering a clear path for participation, we aim to foster a thriving community of designers, developers, and users who work together to ensure the system remains relevant, effective, and resilient over time.
```

Comment:
Should we rename the title of the linked document? e.g resources-contributing-to-the-design-system